.github/workflows/rdev-build-image.yml

```diff
@@ -0,0 +1,63 @@
+name: rdev build-images
```

Comment:
I don't see this being called anywhere, is this an accidental dupe of https://github.com/chanzuckerberg/single-cell-data-portal/blob/main/.github/workflows/build-images.yml ?

Reply:
yes, removed and fixed the commit history.